draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
           </t>
         </section>
       </section>
+
       <section anchor="StreamPriority">
-        <name>Stream Priority</name>
-        <t>
-          A client can assign a priority for a new stream by including prioritization information in
-          the <xref target="HEADERS">HEADERS frame</xref> that opens the stream.  At any other time,
-          the <xref target="PRIORITY">PRIORITY frame</xref> can be used to change the priority of a
-          stream.
-        </t>
-        <t>
-          The purpose of prioritization is to allow an endpoint to express how it would prefer its
-          peer to allocate resources when managing concurrent streams.  Most importantly, priority can
-          be used to select streams for transmitting frames when there is limited capacity for
-          sending.
-        </t>
+        <name>Prioritization</name>
         <t>
-          Streams can be prioritized by marking them as dependent on the completion of other streams
-          (<xref target="pri-depend"/>).  Each dependency is assigned a relative weight, a number
-          that is used to determine the relative proportion of available resources that are assigned
-          to streams dependent on the same stream.
+          In a multiplexed protocol like HTTP/2, prioritizing allocation of resources to streams can
+          be critical to attaining good performance.  A poor prioritization scheme can result in
+          HTTP/2 providing poor performance.  With no parallism at the TCP layer, this could be
+          significantly worse than HTTP/1.1.
         </t>
-        <!--
-          Note that stream dependencies have not yet been validated in practice.  The theory
-          might be fairly sound, but there are no implementations currently sending these.  If it
-          turns out that they are not useful, or actively harmful, implementations will be requested
-          to avoid creating stream dependencies.
-        -->
         <t>
-          Explicitly setting the priority for a stream is input to a prioritization process.  It
-          does not guarantee any particular processing or transmission order for the stream relative
-          to any other stream.  An endpoint cannot force a peer to process concurrent streams in a
-          particular order using priority.  Expressing priority is therefore only a suggestion.
+          A good prioritization scheme benefits from the application of contextual knowledge such as
+          the content of resources, how resources are interrelated, and how those resources will be
```

Comment:
```suggestion
          the content and size of resources, how resources are interrelated, and how those resources will be
```

Reply:
size is just another aspect of content :)

Reply:
Fair enough. It's mainly that size is a parameter typically available only at the server (and is often used in shortest-completion-time schedulers for e.g., multipath) and content (or at least expected content and its semantics/impact) is something typically mainly derived at the client (e.g., difference between render blocking, async or defer JS). 

Now, I don't think it's really necessary to make those nuances here, so I'm fine with dropping it :)